e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -90,13 +90,11 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
           cy.button("Regenerate key");
         });
 
-        cy.findByTestId("-embedded-dashboards-setting").within(() => {
-          cy.findByText(/Embedded dashboards/i);
-          cy.findByText("No dashboards have been embedded yet.");
-        });
-
-        cy.findByTestId("-embedded-questions-setting")
+        cy.findByTestId("-embedded-resources-setting")
           .within(() => {
+            cy.findByText(/Embedded dashboards/i);
```

Comment:
I know you just moved the assertion inside, but can you as well convert from a regex to string? We prefer string when finding texts because it's easier to understand and makes it less prone to error, because regex will also match partial string, and if we really need to match partial strings that's when we would use regexs.